ci/ci/github.py

```diff
@@ -51,6 +51,24 @@ class GithubStatus(Enum):
     FAILURE = 'failure'
 
 
+def github_status(state: str) -> GithubStatus:
+    """
+    Converts a state for a commit status (https://docs.github.com/en/graphql/reference/enums#statusstate)
+    or a conclusion for a check (https://docs.github.com/en/graphql/reference/enums#checkconclusionstate)
+    from the GraphQL API to a GithubStatus.
+    """
+    _state = None
+    if state in {"PENDING", "EXPECTED", "ACTION_REQUIRED", "STALE"}:
+        _state = "pending"
+    elif state in {"FAILURE", "ERROR", "TIMED_OUT", "CANCELLED", "STARTUP_FAILURE", "SKIPPED"}:
+        _state = "failure"
+    elif state in {"SUCCESS", "NEUTRAL"}:
+        _state = "success"
+    if _state is None:
+        raise ValueError(f"Unexpected value for GithubStatus: {state}")
+    return GithubStatus(_state)
```

Comment:
1. Use the enum values, like `GithubStatus.SUCCESS`.
2. Return immediately, rather than setting a value and checking it, like so:
```python3
    if state in {"PENDING", "EXPECTED", "ACTION_REQUIRED", "STALE"}:
        return GithubStatus.PENDING
    if state in {"FAILURE", "ERROR", "TIMED_OUT", "CANCELLED", "STARTUP_FAILURE", "SKIPPED"}:
        return GithubStatus.FAILURE
    # etc.
    raise ValueError(f"Unexpected value for GithubStatus: {state}")
```

Reply:
Done!